apps/windows_terminal/windows_terminal.talon

```diff
@@ -16,6 +16,9 @@ tag(): user.tabs
 # TODO: file_manager
 tag(): user.splits
 
+# Uncomment the following line to enable common unix utilities from unix_utilities.py
+tag(): user.unix_utilities
```

Comment:
I'm not sure whether this should be enabled by default. In my initial pull request, we decided on having this commented in this source code by default. We should stay consistent here, IMO.

Reply:
Good that you pointed that out. I'll fix it once this is final.